modules/auth/sso/reverseproxy.go

```diff
@@ -68,13 +69,22 @@ func (r *ReverseProxy) VerifyAuthData(req *http.Request, w http.ResponseWriter,
 
 	user, err := models.GetUserByName(username)
 	if err != nil {
-		if models.IsErrUserNotExist(err) && r.isAutoRegisterAllowed() {
-			return r.newUser(req)
+		if !models.IsErrUserNotExist(err) || r.isAutoRegisterAllowed() {
+			log.Error("GetUserByName: %v", err)
+			return nil
 		}
-		log.Error("GetUserByName: %v", err)
-		return nil
+		user = r.newUser(req)
 	}
 
+	// Make sure requests to API paths and PWA resources do not create a new session
```

Comment:
Which PWA resources are you referring to? I thought we eliminated all of them (previously `templates/pwa`).

Reply:
I just copied this from the comment on the sspi_windows file.

Reply:
```suggestion
	// Make sure requests to API paths, attachment downloads, git and LFS do not create a new session
```